server/zanata-frontend/src/app/editor/index.html

```diff
@@ -15,7 +15,13 @@
   <meta name="author" content="Red Hat">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="cleartype" content="on">
-
+  <link rel="preconnect" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic" crossorigin />
+  <link rel="preload" as="font" crossorigin type="font/woff"
+        href="../../zanata-war/target/zanata/resources/assets/fonts/zanata-v6/zanata.woff"  />
+  <link rel="preload" as="font" crossorigin type="font/tff"
+        href="../../zanata-war/target/zanata/resources/assets/fonts/zanata-v6/zanata.tff"  />
```

Comment:
Should that be ttf (on both lines)?

Reply:
No, I messed up. Thanks for picking that up.

Reply:
Actually no that is right. Because we need to specify the fallbacks, I need to state what the type of font is also.